### PR TITLE
Add Components

### PR DESCRIPTION
**Pull request by Jawad-Nazari**
**In this PR, I did the following:**

- Created a new branch and deleted all the boilerplate from Create React App
- Created a new directory for the components and added calculator components
- Cleared Linter Errors
- Updated Read me file

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,23 @@
...
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
+          <Calculator />
+        </header>
+      </div>
+    );
+  }
 }
```
Comment (ridwanediallo) [inline src/App.js]: - [ ] You did a good job working on the **App** component. However, one of the requirements says we should use **functional components** for our components. So I would kindly recommend you update this component to a **functional component** instead. :ok_hand: 😊 suggesssion 

### Requirement :point_down: 
Screenshot from 2023-05-02 20-09-34

```
const App = () => {
  return (
    <div className="App">
      <header className="App-header">
        <Calculator />
      </header>
    </div>
  );
};
```

File: src/components/calculator.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            </button>
+            <button className="calc-btn btn-operation" type="button">
+              =
+            </button>
+          </li>
+        </ul>
+      </div>
+    );
+  }
+}
```
Comment (ridwanediallo) [inline src/components/calculator.js]: - [ ] Please also make this **Calculator** component a **functional component**. :ok_hand: 😊

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @Jawad-Nazari , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Your UI design looks nice. ✔
- You did well removing all the boilerplate.  ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js (hunk 2 above)
Comment (Jawad-Nazari) [reply src/components/calculator.js]: Hi @ridwanediallo ,

Thank you for your time and for reviewing my code. I made the Calculator component a functional component. 🙏

File: src/App.js (hunk 1 above)
Comment (Jawad-Nazari) [reply src/App.js]: Hi @ridwanediallo ,

Thank you for your time and for your help. I updated my code with the functional component. 🙏

**Review by ABDULALI3468 — APPROVED** [review]
Hi @Jawad-Nazari , 👍 

# APPROVED 💯 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
☑️Implemented changes asked by the previous reviewer.
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**